### ISSUE TITLE
[BUG]: ImportError: cannot import name 'show_versions'

### ISSUE THREAD

**Issue by mehrdad2000** [pycaret/pycaret#3321]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [x] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

Hi
I got this error:

```
from pycaret import show_versions
show_versions()
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-288b803acbb1> in <module>
----> 1 from pycaret import show_versions
      2 show_versions()

ImportError: cannot import name 'show_versions'
```

Here is the version of package i've installed:
```
Package                   Version
------------------------- -----------
absl-py                   0.15.0
alembic                   1.7.7
anyio                     3.6.2
argon2-cffi               21.3.0
argon2-cffi-bindings      21.2.0
asn1crypto                0.24.0
astunparse                1.6.3
async-generator           1.10
attrs                     22.2.0
Babel                     2.11.0
backcall                  0.2.0
bleach                    4.1.0
blis                      0.7.9
Boruta                    0.3
branca                    0.5.0
cached-property           1.5.2
cachetools                4.2.4
catalogue                 1.0.2
certifi                   2022.12.7
cffi                      1.11.5
chardet                   3.0.4
charset-normalizer        2.0.12
clang                     5.0
click                     8.0.4
cloudpickle               2.2.1
colorlover                0.3.0
configobj                 5.0.6
contextvars               2.4
cryptography              3.2.1
cufflinks                 0.17.3
cycler                    0.11.0
cymem                     2.0.7
Cython                    0.29.33
databricks-cli            0.17.4
dataclasses               0.8
dbus-python               1.2.4
decorator                 4.4.2
defusedxml                0.7.1
Deprecated                1.2.13
docker                    5.0.3
entrypoints               0.4
ethtool                   0.14
file-magic                0.3.0
Flask                     2.0.3
flatbuffers               1.12
folium                    0.13.0
funcy                     1.18
future                    0.18.3
gast                      0.4.0
gensim                    3.8.3
gitdb                     4.0.9
GitPython                 3.1.18
google-auth               1.35.0
google-auth-oauthlib      0.4.6
google-pasta              0.2.0
gpg                       1.13.1
greenlet                  2.0.2
grpcio                    1.48.2
gunicorn                  20.1.0
h5py                      3.1.0
html5lib                  0.999999999
htmlmin                   0.1.12
idna                      3.4
ImageHash                 4.3.1
imageio                   2.15.0
imbalanced-learn          0.7.0
immutables                0.19
importlib-metadata        4.8.3
importlib-resources       5.4.0
iniparse                  0.4
install                   1.3.5
ipykernel                 5.5.6
ipython                   7.16.3
ipython-genutils          0.2.0
ipywidgets                7.7.3
isc                       2.0
itsdangerous              2.0.1
jedi                      0.17.2
Jinja2                    3.0.3
joblib                    1.0.1
json5                     0.9.11
jsonschema                3.2.0
jupyter-client            7.1.2
jupyter-core              4.9.2
jupyter-server            1.13.1
jupyterlab                3.2.9
jupyterlab-pygments       0.1.2
jupyterlab-server         2.10.3
jupyterlab-widgets        2.0.0b0
keras                     2.6.0
Keras-Preprocessing       1.1.2
kiwisolver                1.3.1
kmodes                    0.12.2
libcomps                  0.1.18
lightgbm                  3.3.5
llvmlite                  0.36.0
lxml                      4.2.3
Mako                      1.1.6
Markdown                  3.3.7
MarkupSafe                2.0.1
matplotlib                3.3.4
missingno                 0.5.1
mistune                   0.8.4
mlflow                    1.23.1
mlxtend                   0.19.0
multimethod               1.5
murmurhash                1.0.9
nbclassic                 0.3.5
nbclient                  0.5.9
nbconvert                 6.0.7
nbformat                  5.1.3
nest-asyncio              1.5.6
netifaces                 0.10.6
networkx                  2.5.1
nftables                  0.1
nltk                      3.6.7
notebook                  6.4.10
numba                     0.53.1
numexpr                   2.8.1
numpy                     1.19.3
oauthlib                  3.2.2
opt-einsum                3.3.0
packaging                 21.3
pandas                    1.1.5
pandas-profiling          3.1.0
pandocfilters             1.5.0
parso                     0.7.1
patsy                     0.5.3
pciutils                  2.3.6
perf                      0.1
pexpect                   4.3.1
phik                      0.12.0
pickleshare               0.7.5
Pillow                    8.4.0
pip                       21.3.1
plac                      1.1.3
plotly                    5.13.0
ply                       3.9
pmdarima                  1.8.4
preshed                   3.0.8
prometheus-client         0.16.0
prometheus-flask-exporter 0.22.0
prompt-toolkit            3.0.36
protobuf                  3.19.6
psutil                    5.4.3
ptyprocess                0.5.2
pyasn1                    0.4.8
pyasn1-modules            0.2.8
pycairo                   1.16.3
pycaret                   2.3.10
pycparser                 2.14
pydantic                  1.9.2
pydbus                    0.6.0
Pygments                  2.14.0
PyGObject                 3.28.3
pyinotify                 0.9.6
PyJWT                     2.4.0
pyLDAvis                  3.2.2
pynndescent               0.5.8
pyod                      1.0.7
pyOpenSSL                 19.0.0
pyparsing                 2.1.10
pyrsistent                0.18.0
PySocks                   1.6.8
python-dateutil           2.8.2
python-dmidecode          3.12.2
python-linux-procfs       0.7.0
pytz                      2022.7.1
pyudev                    0.21.0
PyWavelets                1.1.1
PyYAML                    5.4.1
pyzmq                     25.0.0
querystring-parser        1.2.4
regex                     2022.10.31
requests                  2.27.1
requests-oauthlib         1.3.1
rhnlib                    2.8.6
rpm                       4.14.3
rsa                       4.9
schedutils                0.6
scikit-image              0.17.2
scikit-learn              0.24.2
scikit-plot               0.3.7
scipy                     1.5.4
seaborn                   0.11.2
selinux                   2.9
Send2Trash                1.8.0
sepolicy                  1.1
setools                   4.3.0
setroubleshoot            1.1
setuptools                59.6.0
six                       1.16.0
sktime                    0.9.0
slip                      0.6.4
slip.dbus                 0.6.4
smart-open                6.3.0
smmap                     5.0.0
sniffio                   1.2.0
sos                       4.3
spacy                     2.3.9
SQLAlchemy                1.4.46
sqlparse                  0.4.3
srsly                     1.0.6
SSSDConfig                2.7.3
statsmodels               0.12.1
subscription-manager      1.28.32
syspurpose                1.28.32
systemd-python            234
tabulate                  0.8.10
tangled-up-in-unicode     0.1.0
tbats                     1.1.2
tenacity                  8.2.1
tensorboard               2.6.0
tensorboard-data-server   0.6.1
tensorboard-plugin-wit    1.8.1
tensorflow                2.6.2
tensorflow-estimator      2.6.0
termcolor                 1.1.0
terminado                 0.12.1
testpath                  0.6.0
textblob                  0.17.1
thinc                     7.4.6
threadpoolctl             3.1.0
tifffile                  2020.9.3
tornado                   6.1
tqdm                      4.64.1
traitlets                 4.3.3
typing-extensions         3.7.4.3
umap-learn                0.5.3
urllib3                   1.26.14
visions                   0.7.4
wasabi                    0.10.1
Wave                      0.0.2
wcwidth                   0.2.6
webencodings              0.5.1
websocket-client          1.3.1
Werkzeug                  2.0.3
wheel                     0.37.1
widgetsnbextension        3.6.2
wordcloud                 1.8.2.2
wrapt                     1.12.1
yellowbrick               1.3.post1
zipp                      3.6.0
```

any idea?
Thanks

### Reproducible Example

```python
from pycaret import show_versions
show_versions()
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-288b803acbb1> in <module>
----> 1 from pycaret import show_versions
      2 show_versions()

ImportError: cannot import name 'show_versions'
```
```


### Expected Behavior

get version successfully 

### Actual Results

```python-traceback
from pycaret import show_versions
show_versions()
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-288b803acbb1> in <module>
----> 1 from pycaret import show_versions
      2 show_versions()

ImportError: cannot import name 'show_versions'
```
```


### Installed Versions

```
from pycaret import show_versions
show_versions()
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-288b803acbb1> in <module>
----> 1 from pycaret import show_versions
      2 show_versions()

ImportError: cannot import name 'show_versions'
```

**Comment by Yard1** [comment]
`show_versions` is only available in PyCaret 3.0.

**Comment by mehrdad2000** [comment]
@Yard1 try the command that you mention pip install -U --pre pycaret, but it will install pycaret v2.3.10 again.
also try to uninstall and re install it but wont get pycaret PyCaret 3.0.

Any other idea?
Thanks
